### ISSUE TITLE
Analyzers throw TypeLoadException for "Newtonsoft.Json.JsonException"

### ISSUE THREAD

**Issue by RalfKoban** [DotNetAnalyzers/StyleCopAnalyzers#2164]
Hello,

when I compile my code using VS 2015 Update 3, I suddenly get following exception thrown multiple times by the StyleCop analyzers.

```
Severity    Code    Description Project File    Line    Suppression State   Detail Description
Error   AD0001  Analyzer 'StyleCop.Analyzers.DocumentationRules.SA1600ElementsMustBeDocumented' threw an exception of type 'System.TypeLoadException' with message 'Der Typ "Newtonsoft.Json.JsonException" in der Assembly "Newtonsoft.Json, Version=3.5.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed" konnte nicht geladen werden.'. MiKo Aspects        1   Active  Analyzer 'StyleCop.Analyzers.DocumentationRules.SA1600ElementsMustBeDocumented' threw the following exception:
'Exception occurred with following context:
Compilation: MiKoSolutions.Aspects
SyntaxTree: H:\Projekte\MiKo Solutions\MiKo Aspects\trunk\Source\MiKo Aspects\Internals\NDependMarkers\ImmutableAttribute.cs
SyntaxNode: [Pure] [IsNotDeadCode] internal ... [ConstructorDeclarationSyntax]@[556..648) (17,8)-(21,9)

System.TypeLoadException: Der Typ "Newtonsoft.Json.JsonException" in der Assembly "Newtonsoft.Json, Version=3.5.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed" konnte nicht geladen werden.
   at StyleCop.Analyzers.SettingsHelper.GetStyleCopSettings(ImmutableArray`1 additionalFiles, CancellationToken cancellationToken)
   at StyleCop.Analyzers.AnalyzerExtensions.<>c__DisplayClass9_0`1.<RegisterSyntaxNodeActionHonorExclusions>b__0(SyntaxNodeAnalysisContext c)
   at Microsoft.CodeAnalysis.Diagnostics.AnalyzerExecutor.<>c__DisplayClass42_1`1.<ExecuteSyntaxNodeAction>b__1()
   at Microsoft.CodeAnalysis.Diagnostics.AnalyzerExecutor.ExecuteAndCatchIfThrows_NoLock(DiagnosticAnalyzer analyzer, Action analyze, Nullable`1 info)
```

Best regards,
Ralf

**Comment by RalfKoban** [comment]
Sorry for the strange formatting, I'm yet unfamiliar with the Markdown styling and simply pasted the copied exception.

**Comment by sharwell** [comment]
@RalfKoban It's no problem, I updated the formatting :smile:

**Comment by SilverioMiranda** [comment]
I have the same issue here
I have a small project and i get 3000+ errors of this type
![image](https://cloud.githubusercontent.com/assets/1939153/26051101/7f90df66-3937-11e7-8ee9-a8e944182051.png)

**Comment by SilverioMiranda** [comment]
After i downloaded this Repository and installed vsix package needed to build the solution the problem has gone away  as you can see :

![image](https://cloud.githubusercontent.com/assets/1939153/26051756/895a2c12-3939-11e7-89ef-34e6eff72d88.png)

**Comment by sharwell** [comment]
Fixed in release 1.0.2.
